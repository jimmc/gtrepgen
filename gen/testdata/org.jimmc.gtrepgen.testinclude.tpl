Top file, data is {{.}}
No data: {{include "org.jimmc.gtrepgen.included"}}
With foo: {{include "org.jimmc.gtrepgen.included" "foo"}}
End of top file
