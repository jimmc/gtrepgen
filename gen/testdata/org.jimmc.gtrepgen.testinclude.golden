Top file, data is World
No data: This is the included file with data <no value>
With foo: This is the included file with data foo
End of top file
